deploy/gk-deploy

```diff
@@ -308,16 +308,18 @@ END
 )
 
   while read -r node; do
-    debug "Deploying GlusterFS pods on ${node}."
-    if [[ "${CLI}" == *oc* ]]; then
-      ${CLI} process glusterfs GLUSTERFS_NODE=${node} | ${CLI} create -f -
-    else
-      sed -e "s#<GLUSTERFS_NODE>#${node}#" ${TEMPLATES}/glusterfs-deployment.json | ${CLI} create -f -
-    fi
+    debug "Marking '${node}' as a GlusterFS node."
+    ${CLI} label nodes ${node} storagenode=glusterfs
```

Comment:
Sure. I can cover that in a separate PR since there's no good place in the documentation for this just yet.